sdk/core/core-http/src/credentials/accessTokenRefresher.ts

```diff
@@ -34,7 +32,6 @@ export class AccessTokenRefresher {
    * then requests a new token,
    * then sets this.promise to undefined,
    * then returns the token.
-   * @param options -
```

Comment:
There are many more occurrences like this in the repo. Look for `@param .* -$` after enabling the regex search in your editor